store/src/java-test/com/zimbra/cs/db/HSQLDB.java

```diff
@@ -1,7 +1,7 @@
 /*
  * ***** BEGIN LICENSE BLOCK *****
  * Zimbra Collaboration Suite Server
- * Copyright (C) 2011, 2012, 2013, 2014, 2015, 2016 Synacor, Inc.
+ * Copyright (C) 2011, 2012, 2013, 2014, 2015, 2016, 2017 Synacor, Inc.
```

Comment:
I think we should update this information whenever we change the file, unless we have a separate task which does this after every release.